compliance_checker/cf/cf.py

```diff
@@ -203,7 +203,7 @@ def _find_metadata_vars(self, ds, refresh=False):
             elif getattr(var, 'cf_role', None) == 'timeseries_id':
                 self._metadata_vars[ds].append(var)
 
-            elif len(var.dimensions) == 0:
+            elif getattr(var, 'standard_name', None) is None and len(var.dimensions) == 0:
```

Comment:
This has implications. It's hard to know whether a variable is a metadata variable or part of a coordinates system without inspecting feature types.

I'm eliminating some false negatives here by saying if it's dimensionless and it doesn't have a standard_name attribute, it's likely to be an auxiliary, ancillary or metadata variable.